### PR TITLE
refactor(samples): changes with igniteui-theming

### PR DESCRIPTION
**Pull request by simeonoff**

### CODE REVIEW

File: src/_variables.scss [hunk 1]
```diff
@@ -9,8 +9,11 @@ $default-palette: $palette;
 $green-palette: palette(
     $primary: #09f,
     $secondary: #72da67,
-    $grays: #fff,
-    $surface: #333
+    $surface: #333,
```
Comment (didimmova) [inline src/_variables.scss:12]: You're trying to get the gray color shade from this palette but there isn't any gray color passed to the palette.

Comment (simeonoff) [reply src/_variables.scss:12]: That's fine, the gray color is automatically generated when not provided based on the surface color.

**Comment by didimmova** [comment]
- In shadows/card-sample-shadow.scss file the elevations function is being used and it doesn’t exist anymore in the theming package. Are we going to create a function like this there so we can keep using it?

- No surface color in palettes in app/theming/styles files

**Comment by simeonoff** [comment]
@didimmova All version references here need to be updated from 14.2.x. to 15.0.x.

**Comment by didimmova** [comment]
Fix the custom elevations color sample in shadows/card-sample-shadow.scss file

**Comment by simeonoff** [comment]
> 

Done.

**Comment by simeonoff** [comment]
> Fix the custom elevations color sample in shadows/card-sample-shadow.scss file

Done.

**Review by didimmova — DISMISSED** [review]

**Review by desig9stein — APPROVED** [review]

**Pull request merged**